File: reverse_words.py
def reverse_words(string):
 
    words = string.split()

    reversed_words = words[::-1]

    reversed_string = ' '.join(reversed_words)

    return reversed_string

input_string1 = "dkşaslk asdl kasşld aisdk"
input_string2 = "Life is so beautiful"

output_string1 = reverse_words(input_string1)
output_string2 = reverse_words(input_string2)

print(output_string1)
print(output_string2)

File: reverse_words.cs
using System;

class Program
{
    static void Main()
    {
        string inputString1 = "The weather is so sunny nowadays";
        string inputString2 = "Life is so beautiful";

        string outputString1 = ReverseWords(inputString1);
        string outputString2 = ReverseWords(inputString2);

        Console.WriteLine(outputString1);
        Console.WriteLine(outputString2);
    }

    static string ReverseWords(string input)
    {
        string[] words = input.Split(' ');
        Array.Reverse(words);
        return string.Join(" ", words);
    }
}
